dotbim/File.cs

```diff
@@ -30,7 +30,7 @@ public void Save(string path)
             {
                 JsonSerializer serializer = new JsonSerializer
                 {
-                    Formatting = Formatting.Indented
+                    Formatting = Formatting.None
```

Comment:
Hey!

I agree, but probably it should be an optional behaviour in File class, also maybe in File.Save(path, indendent = False) manner to don't break existing behaviour.

Reply:
I think we should make default is true, I think default setting should optimize for developer first.